packages/workbox-routing/_default.mjs

```diff
@@ -54,20 +60,65 @@ class DefaultRouter extends Router {
    */
   registerRoute(capture, handler, method = 'GET') {
     let route;
-    // TODO Should we allow Express Route?
-    // TODO If so - don't forget to add 'string' to params in jsdoc.
-    /** if (typeof capture === 'string') {
-      if (capture.length === 0) {
-        throw new WorkboxError('empty-express-string', {
-          moduleName: 'workbox-routing',
-          class: 'DefaultRouter',
-          func: 'registerRoute',
-          paramName: 'capture',
-        });
+
+    if (typeof capture === 'string') {
+      if (process.env.NODE_ENV !== 'production') {
+        if (!(capture.startsWith('/') || capture.startsWith('http'))) {
+          throw new WorkboxError('invalid-string', {
+            moduleName: 'workbox-routing',
+            className: 'DefaultRouter',
+            funcName: 'registerRoute',
+            paramName: 'capture',
+          });
+        }
+
+        // We want to only prohibit these characters in the pathname portion of
```

Comment:
Can you change this to "We want to check if express style characters are in the pathname only."